test/index.js

```diff
@@ -158,6 +158,8 @@ describe('app', () => {
         ]
       };
       const results = schema.validateAppDefinition(appCopy);
+      // this line ensures we're getting a ValidatorResult class, not just an object that looks like one
+      should(results.valid).eql(true);
```

Comment:
I've confirmed that this will catch regressions: 

```
  1) app
       validation
         should validate inputFormat:
     AssertionError: expected undefined to equal true
      at Assertion.fail (node_modules/should/cjs/should.js:205:17)
      at Assertion.value (node_modules/should/cjs/should.js:277:19)
      at Context.it (test/index.js:161:29)

  2) app
       validation
         should invalidate illegal inputFormat:
     AssertionError: expected undefined to equal false
      at Assertion.fail (node_modules/should/cjs/should.js:205:17)
      at Assertion.value (node_modules/should/cjs/should.js:277:19)
      at Context.it (test/index.js:183:29)
```